qiskit/_compiler.py

```diff
@@ -126,9 +126,11 @@ def compile(circuits, backend,
         if skip_transpiler:  # Just return the qobj, without any transformation or analysis
             job["config"]["layout"] = None
             job["compiled_circuit_qasm"] = circuit.qasm()
+            basis_all = "id,u0,u1,u2,u3,x,y,z,h,s,sdg,t,tdg,rx,ry,rz," \
+                        "cx,cy,cz,ch,crz,cu1,cu3,swap,ccx,cswap"
```

Comment:
I remember seeing many other places where we have similar basis gates definition hardcoded as well, I guess we could review this in the near future and find out a way to have a unique source for it

Reply:
@atilag yes, this definitely has to change. It's ugly and bugs like this pop up. It's on my list to go through all the code and change this. But that's for a future PR.